src/TorchVision/AutoAugment.cs

```diff
@@ -0,0 +1,162 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using static TorchSharp.torch;
+using F = TorchSharp.torchvision.transforms.functional;
+
+#nullable enable
+namespace TorchSharp
+{
+    public static partial class torchvision
+    {
+        private static Tensor apply_op(
```

Comment:
Should this just be a private method in the class? Where else is it used?

Reply:
It will be used by `AutoAugment`, `TrivialAugmentWide`, etc...

Reply:
Time for a base class with a protected member?